zaza/configure/bgp_speaker.py

```diff
@@ -22,6 +26,19 @@ def setup_bgp_speaker(peer_application_name, keystone_session=None):
     :returns: None
     :rtype: None
     """
+    # Get ASNs from deployment
+    dr_relation = ut_juju.get_relation_from_unit(utils.get_juju_model(),
```

Comment:
Can we push the get_juju_model into the utility rather than duplicating it on each call?